src/overlays/actors/ovl_Obj_Kibako/z_obj_kibako.h

```diff
@@ -1,7 +1,7 @@
 #ifndef Z_OBJ_KIBAKO_H
 #define Z_OBJ_KIBAKO_H
 
-#include <global.h>
+#include "global.h"
 
 #define GET_KIBAKO_COLLECTIBLE_ID(x) (((x->actor.params >> 0x8) & 0x7F) << 8)
 #define D_05007890 0x05007890
```

Comment:
Probably outside the scope of this PR, but wanted to mention that this actor has its undefined syms as defines in there `.h` file and should probably be updated.